### ISSUE TITLE
Google Store link 404

### ISSUE THREAD

**Issue by KAR0TTENP0WER** [kapoorlakshya/youtube2music#11]
https://chrome.google.com/webstore/detail/youtube-to-youtube-music/cjcafjnfjeldocjljhejfemlgfogbcjk this link is an error

**Comment by kapoorlakshya** [comment]
Yeah, Google took it down because their new policy forbids extensions from having their product names in it. I need to rename the extension to be able to republish it. Name choices in my head are `yt2music` or `tube2music`.

**Comment by bulletproof2k** [comment]
![Screenshot_2](https://user-images.githubusercontent.com/1818579/191198745-69150633-1ccb-4906-abf0-1ea80ad0674a.png)
